frontend/public/templates/work_packages/tabs/_add_work_package_relation.html

```diff
@@ -9,6 +9,7 @@
     id="relation_to_id-{{ handler.relationsId }}"
     name="relation[to_id][{{ handler.relationsId }}]"
     ng-model="$parent.relationToAddId"
+    title="{{ I18n.t('js.relation_button_inputs.' + handler.relationsId) }}"
```

Comment:
@0xF013 Can we not just make rename the translation keys to conform to relationsId? (I think it's only one key that's actually different)